etc/config/c++.amazon.properties

```diff
@@ -392,6 +392,36 @@ compiler.clang_reflection.semver=(reflection)
 compiler.clang_reflection.options=-std=c++20 -freflection-ts -stdlib=libc++
 compiler.clang_reflection.notification=Experimental Reflection Support; see <a href="https://github.com/cplusplus/reflection-ts" target="_blank" rel="noopener noreferrer">github<sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a>
 
+group.clang-rocm.compilers=clang-rocm-trunk:clang-rocm-40502:clang-rocm-50002:clang-rocm-50103:clang-rocm-50203
+group.clang-rocm.intelAsm=-mllvm --x86-asm-syntax=intel
+group.clang-rocm.options=--gcc-toolchain=/opt/compiler-explorer/gcc-7.2.0
+group.clang-rocm.groupName=ROCm clang x86-64
+group.clang-rocm.instructionSet=amd64
+group.clang-rocm.baseName=x86-64 clang
+group.clang-rocm.isSemVer=true
+group.clang-rocm.compilerType=clang
+group.clang-rocm.unwiseOptions=-march=native
+group.clang-rocm.supportsPVS-Studio=true
```

Comment:
As of a few days ago, we're asking compiler groups to provide license info
`group.clang-rocm.licensePreamble` for custom text to be shown,
`group.clang-rocm.licenseName` for the license name,
`group.clang-rocm.licenseLink` for either a link to the license or other relevant URL.

Can you provide those? All three are optional if you don't have anything for the preamble, for example

Reply:
It's a mix of LLVM Apache 2 and NCSA. There's two links:

https://github.com/RadeonOpenCompute/llvm-project/blob/amd-stg-open/LICENSE.TXT
https://github.com/RadeonOpenCompute/ROCm-Device-Libs/blob/amd-stg-open/LICENSE.TXT

Reply:
I've added the second link in a comment.